pkg/cmd/server/api/conversions.go

```diff
@@ -1,15 +0,0 @@
-package api
-
-func init() {
-	Scheme.AddDefaultingFuncs(
```

Comment:
I didn't catch this in the previous review.  Defaulters belong to versions.  This would need to be in the `v1/conversions.go` with the rest of the defaulters.